osu.Game/Overlays/Rankings/Tables/CountriesTable.cs

```diff
@@ -70,15 +71,15 @@ private class CountryName : LinkFlowContainer
             [Resolved(canBeNull: true)]
             private RankingsOverlay rankings { get; set; }
 
-            public CountryName(Country country)
+            public CountryName(CountryCode countryCode)
                 : base(t => t.Font = OsuFont.GetFont(size: 12))
             {
                 AutoSizeAxes = Axes.X;
                 RelativeSizeAxes = Axes.Y;
                 TextAnchor = Anchor.CentreLeft;
 
-                if (!string.IsNullOrEmpty(country.FullName))
-                    AddLink(country.FullName, () => rankings?.ShowCountry(country));
+                if (countryCode != CountryCode.Unknown)
+                    AddLink(countryCode.GetDescription(), () => rankings?.ShowCountry(countryCode));
```

Comment:
Not 100% on the use of `GetDescription` to get the country name. It feels a bit too generic of a method call? Guess it works for now though. If it ever comes up as an issue we could make an extension method to use instead. Or maybe reconsider whether `enum` is the best fit.